.teamcity/src/main/kotlin/configurations/DocsTest.kt

```diff
@@ -119,7 +119,7 @@ class DocsTest(
     testClasses: List<String>,
 ) : BaseGradleBuildType(stage = stage, init = {
 
-    id("${model.projectId}_${docsTestType.docsTestName}_${testJava.version.name.toCapitalized()}_${os.asName()}_$index")
+    id("${model.projectId}_${docsTestType.docsTestName}_${os.asName()}_$index")
```

Comment:
❔ I am not sure if we run the docs tests on different Java versions. If we do, we probably shouldn't remove it from the name and do something else instead.